CreatubblesAPIClient/Sources/Mapper/NotificationMetadataMapper.swift

```diff
@@ -27,11 +27,15 @@ import UIKit
 import ObjectMapper
 
 class NotificationMetadataMapper: Mappable {
+    var totalNewCount: Int?
```

Comment:
It'd be helpful to add a comment stating what each field means e.g. `new` vs `unread`. Is `unread` a notification that has been opened e.g. go to details or one that hasn't been displayed on a list?